examples/nextjs/package.json

```diff
@@ -14,7 +14,7 @@
 		"next": "14.0.4",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
-		"zod": "3.25.3"
+		"zod": "3.25.30"
```

Comment:
I think I lock the zod version previously because of the preprocess bug in zod. It should be find to use a caret range now, e.g. `^3.25.30`.